mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
having both `delays` and `times` seem redundant no?

yeah, sklearn API requires that the parameter are not between init and storage. What you could do however, is make `times` a dynamic attibrute which is constructed from the `tmin`, `tmax` & `sfreq` param, in which case no underscore is needed.

A good way to check this in the test is to try sklearn.base.clone your estimator and check that it is identical.